examples/observability/eks-cluster-with-amp-amg-opensearch/main.tf

```diff
@@ -234,15 +235,8 @@ resource "aws_elasticsearch_domain_policy" "opensearch_access_policy" {
 }
 
 resource "aws_security_group" "opensearch_access" {
-  vpc_id = module.aws_vpc.vpc_id
-
-  ingress {
```

Comment:
was for the Bastion route - no longer a recommended practice to use this route but SSM Session Manger instead if needed